services/consensuscontext/validate_transactions_block_test.go

```diff
@@ -23,15 +23,27 @@ import (
 )
 
 func toTxValidatorContext(cfg config.ConsensusContextConfig) *txValidatorContext {
-	block := builders.BlockPairBuilder().Build()
-	prevBlockHashCopy := make([]byte, 32)
+	return toTxValidatorContextWithBc(cfg, false)
+}
+
+func toTxValidatorContextWithBc(cfg config.ConsensusContextConfig, isBc bool) *txValidatorContext {
```

Comment:
Please use full names - what's `isBc`?

Reply:
changed name to isBackwardsCompatible -> creats the block with an empty hash (len=0) block-proposer to simulate an older version